.eslintrc

```diff
@@ -14,3 +14,13 @@ plugins:
 rules:
   no-console: 0  # This project purposely uses console.
   prefer-reflect: 0
+  jsdoc/check-param-names: 1
```

Comment:
They only work with the appropiate JSdoc-eslint plugin installed so it's quite project specific.

https://github.com/gajus/eslint-plugin-jsdoc